exercises/practice/prime-factors/.meta/example.cairo

```diff
@@ -0,0 +1,20 @@
+pub fn factors(n: u64) -> Array<u64> {
+    let mut all_prime = ArrayTrait::new();
+
+    let mut num: u64 = n;
+    loop {
+        for i in 2..(num + 1) {
```

Comment:
Although this works, it seems kind of redundant to continuously recreate a for loop + range that goes to `num + 1`, but then almost never reach it. 
Example solutions are there to showcase an idiomatic way to solve the problem for a given language, so we need to pay attention to clean code.
There is a way to write this function without depending on breaks and never-to-be-reached ranges, try to rewrite it in such a way. You can use [Rust's](https://github.com/exercism/rust/blob/main/exercises/practice/prime-factors/.meta/example.rs) implementation as inspiration.